src/main/java/uk/gov/ons/ctp/integration/rhsvc/representation/WebformPersistedDTO.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.ons.ctp.integration.rhsvc.representation;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.io.Serializable;
+import java.util.Date;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.ons.ctp.common.jackson.CustomDateSerialiser;
+
+/** This object holds webform data with extra details for the datastore. */
+@Data
+@NoArgsConstructor
+@SuppressWarnings("serial")
+public class WebformPersistedDTO implements Serializable {
```

Comment:
remove SuppressWarnings and implementing Serializable. Not needed.

Reply:
Removed